### ISSUE TITLE
Error when query >=1000 records

### ISSUE THREAD

**Issue by vzenix** [neo4j-php/neo4j-php-client#155]
**Describe the bug**
The library fails when the result have >=1000 records

**To Reproduce**
Query >= records and use AbstractCypherSequence->toArray()

**Expected behavior**
Get records

**Screenshots**
It's back code, i have not a screenshot.

**Desktop (please complete the following information):**
 - Library version: ^2.7
 - Neo4j Version: 4.*
 - PHP version: 8.0
 - OS: Dockerfile
```
# pick the image we used before as our starting place
FROM php:8.0-apache

# Run apt-get to install our packages
RUN apt-get update && apt-get install -qq --no-install-recommends \
    cron \
    libzip-dev \
    zip \
    openssl\
 && docker-php-ext-install opcache sockets zip\
 && a2enmod ssl\
 && a2enmod rewrite\
 && a2enmod headers\
 && apt-get clean all

# etc.
```

**Additional context**
Previous versions have not this error

Exception

```
Objeto: ErrorException
Mensaje: Warning: Undefined array key 999
Codigo de la excepcion: 0
Linea: 485
Binario: /var/www/html/vendor/laudis/neo4j-php-client/src/Types/AbstractCypherSequence.php
Trace:#0 /var/www/html/vendor/laudis/neo4j-php-client/src/Types/AbstractCypherSequence.php(435): Laudis\Neo4j\Types\AbstractCypherSequence->cacheKey()
#1 /var/www/html/vendor/laudis/neo4j-php-client/src/Types/ArrayList.php(54): Laudis\Neo4j\Types\AbstractCypherSequence->current()
#2 [internal function]: Laudis\Neo4j\Types\ArrayList::Laudis\Neo4j\Types\{closure}()
#3 /var/www/html/vendor/laudis/neo4j-php-client/src/Types/AbstractCypherSequence.php(457): Generator->next()
#4 /var/www/html/vendor/laudis/neo4j-php-client/src/Types/AbstractCypherSequence.php(87): Laudis\Neo4j\Types\AbstractCypherSequence->next()
#5 /var/www/html/vendor/laudis/neo4j-php-client/src/Types/ArrayList.php(54): Laudis\Neo4j\Types\AbstractCypherSequence->Laudis\Neo4j\Types\{closure}()
#6 [internal function]: Laudis\Neo4j\Types\ArrayList::Laudis\Neo4j\Types\{closure}()
#7 /var/www/html/vendor/laudis/neo4j-php-client/src/Types/AbstractCypherSequence.php(457): Generator->next()
#8 /var/www/html/vendor/laudis/neo4j-php-client/src/Types/AbstractCypherSequence.php(536): Laudis\Neo4j\Types\AbstractCypherSequence->next()
#9 /var/www/html/vendor/laudis/neo4j-php-client/src/Types/AbstractCypherSequence.php(401): Laudis\Neo4j\Types\AbstractCypherSequence->preload()
#10 /var/www/html/lib/sf-lib-neo4j-bundle/Common/Neo4j/RecordHelper.php(96): Laudis\Neo4j\Types\AbstractCypherSequence->toArray()
```

**Comment by stefanak-michal** [comment]
I think it's duplicate for #149

**Comment by transistive** [comment]
Fixed in dev-main and will be released in 2.8.1 later today
